src/Tests/Tests.ValueArgument.cs

```diff
@@ -37,6 +40,42 @@ public CommandLineParser.CommandLineParser InitValueArgument()
             return commandLineParser;
         }
 
+        [Fact]
+        public void ValueArgumentWithOptionalValue_shouldReturnDefaultValue_whenParameterNotPassed_andForcedDefaultValueNotSet()
+        {
+            // ARRANGE 
+            string[] args = { " " };
+            var commandLineParser = InitValueArgument();
+            // ACT 
+            commandLineParser.ParseCommandLine(args);
+            // ASSERT
+            Assert.Equal(9, valueArgumentTarget.Duration);
```

Comment:
Maybe also add
`Assert.Equal(?, valueArgumentTarget.Version);` to test the `Version` which uses ForcedDefaultValue?

Reply:
That would be correct, but the test would check multiple cases.
In my understanding this is already tested here :
https://github.com/j-maly/CommandLineParser/blob/cadd7d11d4aa56ffbfe569dd89ff3214ee5f3ff8/src/Tests/Tests.ValueArgument.cs#L186